src/Glimpse.Common/Broker/Message.cs

```diff
@@ -9,14 +9,16 @@ public class Message : IMessage
     {
         public Guid Id { get; set; }
 
-        public string Type { get; set; }
+        public IEnumerable<string> Types { get; set; }
 
+        [JsonConverter(typeof(RawStringConverter))]
         public string Payload { get; set; }
 
-        public MessageContext Context { get; set; }
+        public int Ordinal { get; set; }
 
-        public IEnumerable<string> Tags { get; set; }
+        public MessageContext Context { get; set; }
 
+        [JsonIgnore]
```

Comment:
since payload is already converted, I wonder if we should just manually serialize this type? Probably not worth the perf difference?